tox.ini

```diff
@@ -10,6 +10,37 @@ requires =
 tox_pip_extensions_ext_venv_update = true
 tox_pyenv_fallback = false
 
+[pycodestyle]
+ignore =
+    # Disable pycodestyle errors and warnings that we don't care about because
+    # Black formats our code for us.
+    E203,  # Whitespace before ':',
+    E231,  # Missing whitespace after ',',
+    E501,  # Line too long,
+    W503,  # Line break before binary operator,
+
+    # Bare except. PyLint finds these for us so we don't need pycodestyle to.
+    E722,
+
+[pydocstyle]
+ignore =
+    # Missing docstrings.
+    D100,D101,D102,D103,D104,D105,D106,D107,
+
+    # "No blank lines allowed after function docstring" conflicts with
+    # the Black code formatter, which insists on inserting blank lines after
+    # function docstrings.
+    D202,
+
+    # "1 blank line required before class docstring" conflicts with another
+    # pydocstyle rule D211 "No blank lines allowed before class docstring".
+    D203,
+
+    # Multi-line docstring summary should start at the first line, this
+    # conflicts with another pycodestyle rule "Multi-line docstring summary
+    # should start at the second line".
+    D212,
```

Comment:
The pydocstyle config is in here so that it's in the same place as the pycodestyle config